src/drafts.ts

```diff
@@ -6,53 +6,56 @@
 import * as config from 'config'
 import * as UUID from 'uuid/v4'
 
-import {JsonRpcError, JsonRpcMethodContext} from '@steemit/koa-jsonrpc'
+import {JsonRpcAuthMethodContext as JCtx, JsonRpcError} from '@steemit/koa-jsonrpc'
 import {store} from './store'
 
 const KEY_PREFIX = config.get('name')
-function draftsKey(username: string) {
-    return `${ KEY_PREFIX }_${ username }_drafts.json`
+function draftsKey(account: string) {
+    return `${ KEY_PREFIX }_${ account }_drafts.json`
 }
 
-async function readDrafts(username: string): Promise<any[]> {
-    return await store.readJSON(draftsKey(username)) || []
+async function readDrafts(account: string): Promise<any[]> {
+    return await store.readJSON(draftsKey(account)) || []
 }
 
-async function writeDrafts(username: string, drafts: any) {
-    await store.writeJSON(draftsKey(username), drafts)
+async function writeDrafts(account: string, drafts: any) {
+    await store.writeJSON(draftsKey(account), drafts)
```

Comment:
`return await`

Reply:
the missing return is not accidental here, we don't want to return the store metadata to users